scripts/rs-enum.sh

```diff
@@ -94,67 +94,80 @@ if [ -n "${mdev}" ]; then
 dot=$(${media_util} -d ${mdev} --print-dot)
 
 vid_dev_idx=$(echo "${dot}" | grep "DS5 mux" | grep "vi-output" | tr '\\n' '\n' | grep video | awk -F'"' '{print $1}' | grep -Po "\\d+")
+
 [[ -z ${vid_dev_idx} ]] && exit 0
+
+# get cameras physical location
+dfuid=($(ls -1 /sys/class/d4xx-class/ | awk -F'-' '{print $4}'))
```

Comment:
Can you add in a comment how do you expect the output to look like?
Which work are you interested in?
Might help to understand better in the future

Reply:
this is self explanatory
```
$ ls -1 /sys/class/d4xx-class/
d4xx-dfu-30-001a
```